public/css/index.css

```diff
@@ -5,6 +5,14 @@
     box-sizing: border-box;
 }
 
+.body {
```

Comment:
Just remove the dot and it will behave correctly, also remove the class in the html file, it's unnecessary.